db/schema.rb

```diff
@@ -212,7 +212,7 @@
     t.integer  "client_data_id"
     t.string   "client_data_type", limit: 255
     t.integer  "requester_id"
-    t.string   "public_id",        limit: 255
```

Comment:
I think this keeps happening because I am running my migrations against a copy of the prod db, which has the explicit `limit` declared.